src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,37 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
```

Comment:
You don't need to send in every constructor parametr "health = 100", because
1. You don't send this parametr in arguments
2.If you write just `this.health = 100`, you can write this once, in Animal class, and doesn't rewrite this else

Reply:
Yeah, you are right!

Reply:
@Jamal-Amoian Yeah, you are right!